examples/amp-consent/cmp-vendors.amp.html

```diff
@@ -185,6 +186,17 @@ <h3>Image that is NOT blocked by consent</h3>
         </amp-consent>
         <!-- End didomi example -->
 
+      <!-- iubenda example -->
+      <amp-consent id="iubenda" layout="nodisplay" type="iubenda">
+        <script type="application/json">
+          {
+            "consentRequired": true,
```

Comment:
Also, I would remove `"consentRequired": true,` as it might be a little confusing.

Reply:
Done.